app/blueprints/assessments/forms/assignment_forms.py

```diff
@@ -41,8 +44,12 @@ def validate(self, extra_validators=None):
                 # From a redirect, not a form post
                 return False
 
-            if not request.form.getlist("selected_users"):
-                self.form_errors.append("At least one assessor should be selected")
+            if set(request.form.getlist("assigned_users")) == set(
```

Comment:
to get rid of else 
if set(request.form.getlist("assigned_users")) != set(
                request.form.getlist("selected_users")
            ):
                 return True

self.form_errors.append(
    "No changes have been made to the current assignments for this application"
)
return False